gui/dmg_settings.py

```diff
@@ -0,0 +1,154 @@
+import os.path
```

Comment:
This looks like a generated file, right? We need a license header if not.

Reply:
It is not. I checked however, and it doesn't look like any files in the repo have a license header. I also checked that it is OK to only have repo-level LICENSE file and stick with that if that's the current convention in the repo.